Gemfile.lock

```diff
@@ -332,5 +332,8 @@ DEPENDENCIES
   webmock (~> 3.0.1)
   yard
 
+RUBY VERSION
```

Comment:
It would be good to not have this unrelated change

Reply:
We left it because it was a small change with little impact, but we can remove it if you think it's necessary.
Were you thinking of putting it in a different commit or removing it completely from this PR @barrucadu ?

Reply:
I think just remove it entirely

Reply:
Can do